foreign_cc/cmake.bzl

```diff
@@ -197,6 +195,13 @@ def _get_generator_target(ctx):
         else:
             fail("No generator set and no default is known. Please set the cmake `generator` attribute")
 
+    if "Unix Makefiles" == generator:
```

Comment:
I feel like this is unnecessary and you just append the generator args in the `_cmake_impl` macro. Either that or move allow this macro to also return `tool_deps` or something so there's only one where we check generator info. I think I prefer the latter

Reply:
Good shout that's a better idea than this, it didn't seem right when I added it.